salt/modules/lxc.py

```diff
@@ -2150,8 +2157,22 @@ def _change_state(cmd, name, expected):
         __salt__['cmd.run']('lxc-stop -k -n {0}'.format(name),
                             python_shell=False)
 
+    # certain lxc commands need to be taken with care (lxc-start)
+    # as te command itself mess with double forks; we must not
+    # communicate with it, but just wait for the exit status
+    pkwargs = {'python_shell': False,
+               'with_communicate': with_communicate,
+               'use_vt': 'use_vt',
```

Comment:
Should this be `'use_vt': use_vt,` or was it intentionally a string that was passed in?